x/mint/types/params.go

```diff
@@ -21,7 +21,11 @@ var (
 	DefaultMaxBlockDuration      time.Duration      = time.Minute
 	DefaultFeeCollectorRecipient InflationRecipient = InflationRecipient{
 		Recipient: authtypes.FeeCollectorName,
-		Ratio:     sdk.OneDec(),
+		Ratio:     sdk.MustNewDecFromStr("0.80"), // 80%
+	}
+	DefaultRewardsRecipient InflationRecipient = InflationRecipient{
+		Recipient: "rewards",
```

Comment:
I'm not sure if this should be here without explicit dependency on x/rewards. 

I don't think that making x/mint depend on x/rewards is a good idea too.